sigs.yaml

```
@@ -1676,7 +1676,7 @@ sigs:
     time: "18:30"
     tz: UTC
     frequency: biweekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoom-link
```

```suggestion
    url: https://zoom.us/j/577372069
```
from agenda here: https://docs.google.com/document/d/13uHgcLf-hcR4a5QbV888fhnVsF3djBEpN8HolwS0kWM/edit